docs/en/government.md

```diff
@@ -67,3 +67,9 @@ Category
 ### Dispute settlement news
 
 <RouteEn author="nczitzk" example="/wto/dispute-settlement" path="/wto/dispute-settlement/:year?" :paramsDesc="['Year, current year by default']"/>
+
+## Hong Kong Independent Commission Against Corruption
+
+### Press Releases
+
+<Route author="linbuxiao" example="/icac/news/sc" path="/icac/news/:lang?" :paramsDesc="['Language, default to `sc`. Supprot `en`(English), `sc`(Simplified Chinese) and `tc`(Traditional Chinese)']">
```

Comment:
```suggestion
<RouteEn author="linbuxiao" example="/icac/news/sc" path="/icac/news/:lang?" :paramsDesc="['Language, default to `sc`. Supprot `en`(English), `sc`(Simplified Chinese) and `tc`(Traditional Chinese)']"/>
```

Reply:
This should fix the issue of route not searchable in https://docs.rsshub.app

Reply:
I will make changes to this in the next PR~

Reply:
pr #8273